src/pages/Profile.tsx

```diff
@@ -31,13 +33,48 @@ const GeneralDetailsAboutLine = () => {
   )
 }
 
+const LineProfileHeader = ({
+  operator_ref,
+  agency_name,
+  route_short_name,
+  route_long_name,
+}: {
+  operator_ref: number
+  agency_name: string
+  route_short_name: string
+  route_long_name: string
+}) => {
+  const { t } = useTranslation()
+  const splitRouteName = route_long_name.split('<->')
+  return (
+    <>
+      <OperatorCard>
+        <img width={'60px'} height={'60px'} src={`../operators-logos/${operator_ref}.svg`} />
+        <Typography variant="h6" sx={{ fontWeight: 'bold' }}>
+          {agency_name}
+        </Typography>
+      </OperatorCard>
+      <HeaderContainer>
+        <h2 className="title">
+          {t('lineProfile.title')} {route_short_name}
+        </h2>
+        <div style={{ display: 'flex', alignItems: 'center', gap: '5px' }}>
+          {' '}
+          {splitRouteName[0]} <MultipleStopOutlined /> {splitRouteName[1]}{' '}
+        </div>
```

Comment:
It is not a good practice to style things via inline styling. 
I think it would be better via CSS (or any other CSS solution the project is using)

Reply:
updated, thanks!